src/pages/workspace/companyCards/addNew/AddNewCardPage.tsx

```diff
@@ -7,13 +7,16 @@ import CardInstructionsStep from './CardInstructionsStep';
 import CardNameStep from './CardNameStep';
 import CardTypeStep from './CardTypeStep';
 import DetailsStep from './DetailsStep';
+import SelectBankStep from './SelectBankStep';
 
 function AddNewCardPage() {
     const [addNewCardFeed] = useOnyx(ONYXKEYS.ADD_NEW_COMPANY_CARD);
 
     const {currentStep} = addNewCardFeed ?? {};
 
     switch (currentStep) {
+        case CONST.COMPANY_CARDS.STEP.SELECT_BANK:
+            return <SelectBankStep />;
```

Comment:
How will this work with when the rest of the flow is not done yet? The direct feed options need to go behind beta, there is the canUseDirectFeeds option. I think we should only show the custom feeds options unless user has the direct feeds beta

Reply:
Lets make sure that we only support the custom feed after this change since there is no way to connect to the api yet